.env.example

```diff
@@ -47,6 +50,16 @@ GATSBY_DURATION_GRANT_TIER4 = 1200
 GATSBY_DURATION_GRANT_TIER5 = 1200
 GATSBY_DURATION_GRANT_TIER6 = 1200
 
+# Duration for governance process proposals
+GATSBY_DURATION_PRE_PROPOSAL_POLL=1200
```

Comment:
Make it shorter and consistent:
GATSBY_DURATION_POLL
GATSBY_DURATION_DRAFT
GATSBY_DURATION_GOVERNANCE

This way it matches with the naming of the threshold. GATSBY_SUBMISSION_THRESHOLD_POLL/DRAFT/GOVERNANCE